official-stack/spring-vue/README.md

```diff
@@ -0,0 +1,8 @@
+# Spring-Vue Stack
+
+## Features
+- Spring Boot
+- Vue
+
+## Documentation
+- [Stack documentation](https://heighliner.dev/docs/core_features/stacks/overview)
```

Comment:
Point this to the Spring-Vue tutorial: https://heighliner.dev/docs/tutorials/spring_vue